protocols/kad/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.46.0 - unreleased
```

Comment:
Should this really be a breaking change? It is changing behaviour but arguably to better defaults!